.github/workflows/coverage-linux.yml

```
@@ -3,67 +3,81 @@ name: Coverage Linux
+  CXX: sccache clang++
+  SCCACHE_GHA_ENABLED: 'true'
 
 permissions:
   contents: read
 
 jobs:
   coverage-linux:
-    if: github.event.pull_request.draft == false
-    runs-on: ubuntu-latest
+    # Disabled because "Report JS" step was crashing.
+    if: false
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)